src/app/app.module.ts

```diff
@@ -11,7 +11,7 @@ import { MatInputModule } from '@angular/material/input';
 import { MatTabsModule } from '@angular/material/tabs';
 import { MatToolbarModule } from '@angular/material/toolbar';
 
-import { MdePopoverModule } from '@jaguards/material-extended-mde';
+import { MdePopoverModule } from '../../projects/material-extended/mde/src/lib/popover';
```

Comment:
remettre l'import de la lib et pas la local 

```suggestion
import { MdePopoverModule } from '@jaguards/material-extended-mde';
```